<launch>
  <node name="ms" pkg="map_server" type="map_server" args="$(find costmap_2d)/test/bigmap.yaml"/>
  <test time-limit="300" test-name="static_with_inflation_tests" pkg="costmap_2d" type="static_with_inflation_tests" launch-prefix="">
    <param name="inflation/cost_scaling_factor" value="1" />
  </test>
</launch>
